CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -80,7 +80,7 @@ strDropUpgrade=REMOVE UPGRADE
 strStripUpgrades=MAKE UPGRADES AVAILABLE
 strStripUpgradesTooltip=Remove all weapon upgrades from primary weapons NOT equipped by soldiers selected for current mission duty.
 strStripUpgradesConfirm=CONFIRM WEAPON UPGRADES AVAILABLE
-strStripUpgradesConfirmDesc=Confirm removal of all weapon upgrades from all primary weapons NOT equipped by soldiers for current mission duty. These weapon upgrades will become available immediately.
+strStripUpgradesConfirmDesc=Confirm removal of all weapon upgrades from all primary weapons NOT equipped by soldiers for current mission duty. These weapon upgrades will become available immediately. Upgrades from weapons with nicknames WILL be removed by default. This can be changed in MCM.

```

Comment:
I think it would be much better to show to player which option is enabled currently, rather than saying "by default"